tests/catalog/test_db.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session
+from datetime import datetime
+from recap.catalogs.db import DatabaseCatalog, CatalogEntry
+
+class TestCatalogEntry:
+    def test_is_deleted(self):
+        entry = CatalogEntry(deleted_at=datetime.now())
+        assert entry.is_deleted() == True
+
+        entry = CatalogEntry(deleted_at=None)
+        assert entry.is_deleted() == False
+
+class TestDatabaseCatalog:
```

Comment:
Nit: I've been doing 2 blank lines between classes to play nicely with linters.

Reply:
In other projects I usually use black and isort for code style.
Should we open an issue to discuss linters and code formatting or is it to soon?

Reply:
Yea, I think it's worth opening an issue for it. Sooner the better.